### PR TITLE
Updated readme example to @most/core

### PR DESCRIPTION
**Pull request by kanitsharma**
Noticed this while I was checking the library out. :smiley:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 
 ```js
 import { click } from '@most/dom-event';
+import { tap runEffects } from "@most/core";
+import { newDefaultScheduler } from "@most/scheduler";
 
 const clickStream = click(el);
 
-clickStream.observe(event => console.log(event));
+runEffects(tap(console.log, clickStream), newDefaultScheduler());
```
Comment (briancavalier) [inline README.md:12]: :+1: thanks!

File: README.md [hunk 2]
```diff
@@ -20,10 +22,10 @@ clickStream.observe(event => console.log(event));
 
 #### &lt;eventName&gt; :: (EventTarget t, Event e) => t &rarr; boolean=false &rarr; Stream e
 
-See [the source](src/index.js) for all the supported event names.  Each has the general signature:
+See [the source](src/index.js) for all the supported event names. Each has the general signature:
 
 ```js
-const stream = eventName(domNode, useCapture = false)
+const stream = eventName(domNode, (useCapture = false));
```
Comment (briancavalier) [inline README.md]: Hmm, I think this was just a confusing way to try to document these functions to start with.  How about this: Undo the addition of the parens to this line, and the similar `domEvent` line below, and I'll change how we documented this to a style that's similar to the `@most/core` docs.  Sound ok?

**Review by briancavalier — CHANGES_REQUESTED** [review]
Looks good.  Thanks!  Sorry for the confusing way the default useCapture param value was documented.

File: README.md (hunk 2 above)
Comment (kanitsharma) [reply README.md]: Sure :+1:

File: README.md [hunk 3]
```diff
@@ -1,13 +1,15 @@
 # most dom-event
 
-Streamlined DOM events for [@most/core](https://github.com/mostjs/core).  Now you can write:
+Streamlined DOM events for [@most/core](https://github.com/mostjs/core). Now you can write:
 
 ```js
 import { click } from '@most/dom-event';
+import { tap runEffects } from "@most/core";
```
Comment (briancavalier) [inline README.md]: Just noticed this needs a comma between `tap` and `runEffects`.

**Review by briancavalier — CHANGES_REQUESTED** [review]
Just a missing comma, then this looks good to merge!

File: README.md (hunk 3 above)
Comment (kanitsharma) [reply README.md]: shit man :man_facepalming:

**Review by briancavalier — APPROVED** [review]

**Comment by briancavalier** [comment]
Thanks @kanitsharma!

**Pull request merged**